common/pop_jobs/hfe_job_population.txt

```diff
@@ -39,26 +76,59 @@ hfe_job_population_navigator.1 = {
 	building_icon = building_psi_corps
 	clothes_texture_index = 4
 	icon = hfe_job_population_navigator.0
+	possible_pre_triggers = {
+		is_enslaved = no
+#		The Navigator Academy is an elite institution, it will not admit lowly slaves
+	}
 	possible = {
-		or = {
-			has_job = hfe_job_population_navigator.1
-			hfe_trigger_uncapped_job = yes
-		}
+#		or = {
+#			has_job = hfe_job_population_navigator.1
+#			hfe_trigger_uncapped_job = yes
+#		}
 		exists = owner
-		hfe_trigger_is_psionic = yes
-		worker_job_check_trigger = yes
+#		hfe_trigger_is_psionic = yes
```

Comment:
must be psionic

Reply:
You are bringing up an issue we already solved and you already accepted. This complaint was filed once before. You do not need to be Psionic to use Zro. Psionic Pops always get the Job if they exist.

You need to stop bringing up issues we already dicussed.

Reply:
This feature intended only for psi empires

Reply:
Only Psionic Empires can discover the Navigator's Guild. Other nations can learn of it by being friend with a Psionic Empire. They cannot unlock the technology themselves.